nix/haskell.nix

```diff
@@ -0,0 +1,41 @@
+# This file is part of the IOGX template and is documented at the link below:
+# https://www.github.com/input-output-hk/iogx#32-nixhaskellnix
+
+{ system, ... }:
+
+{
+  supportedCompilers = [ "ghc8107" ];
+
+
+  enableCrossCompilation = true;
+
+
+  defaultChangelogPackages = [ ];
+
+  # Because of lack of resources in our darwin CI cluster, the "combined-plutus-apps-haddock" job
+  # often gets timed out, which of course makes the required darwin CI checks fail. In the end, we
+  # just remove it from the required darwin jobs, but still keep it in the linux jobs.
+  enableCombinedHaddock = system == "x86_64-linux";
+
+
+  projectPackagesWithHaddock = [
+    "plutus-core"
```

Comment:
These were copied from plutus.
Which of `plutus-apps`'s cabal projects have haddock in them?